src/omnibase_core/validators/circular_import_validator.py

```
@@ -128,36 +133,38 @@ def _test_import(self, module_name: str, file_path: Path) -> ModuleImportResult:
-                return ModuleImportResult(
+                return ModelModuleImportResult(
                     module_name=module_name,
-                    status=ImportStatus.IMPORT_ERROR,
+                    status=EnumImportStatus.IMPORT_ERROR,
                     error_message=error_msg,
                     file_path=str(file_path),
                 )
 
-        except Exception as e:
+        except (
+            Exception
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)